templates/controller-mutatingwebhookconfiguration.yaml

```diff
@@ -21,9 +21,12 @@ webhooks:
       namespace: {{ .Release.Namespace }}
       path: /mutate-v1alpha1-proxydefaults
   failurePolicy: Fail
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1" }}
```

Comment:
NOTE: We will need to update our contributing docs for CRDs as well: https://github.com/hashicorp/consul-k8s/blob/master/CONTRIBUTING.md#updating-consul-helm